types/xrm/index.d.ts

```diff
@@ -2488,7 +2478,7 @@ declare namespace Xrm {
         /**
          * The name
          */
-        name?: string | undefined;
```

Comment:
Is this intentionally removed? Usually `| undefined` is needed for `exactOptionalPropertyTypes`.